packages/breadcrumbs/src/styled/StyledBreadcrumb.ts

```diff
@@ -6,19 +6,24 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import BreadcrumbStyles from '@zendeskgarden/css-breadcrumbs';
-import { retrieveComponentStyles, isRtl } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'breadcrumbs.list';
 
-export const StyledBreadcrumb = styled.ol.attrs(props => ({
-  'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(BreadcrumbStyles['c-breadcrumb'], {
-    // RTL
-    [BreadcrumbStyles['is-rtl']]: isRtl(props)
-  })
-}))`
-  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
+/**
+ * 1. <ol> reset.
+ */
+export const StyledBreadcrumb = styled.ol.attrs(() => {
+  return {
+    'data-garden-id': COMPONENT_ID,
+    'data-garden-version': PACKAGE_VERSION
+  };
+})`
+  ${props => `
+    display: flex; ${/* [1] */ ''}
```

Comment:
Comments can be inline (no template strings). They are removed in production builds with the babel plugin.

```
display: flex; /* [1] */
```